### PR TITLE
feat: file in instances

### PR DESCRIPTION
**Pull request by dominikhorn93**
**Description**
Add functionality to call instance level file endpoints

**Reference**
Issues (internal)

**Checklist**

- [ ] Acceptance criteria are met
- [x] Pipeline has been run successfully
- [ ] JUnit tests have been run successfully
- [x] Changelog is adjusted
- [ ] [Documentations](https://git.muenchen.de/groups/digitalisierung/-/wikis/Dokumentationen) are completed
- [x] Frontend is tested
- [ ] Created sub-branches are deleted
- [ ] Boards are updated ( [Digitalisierung](https://git.muenchen.de/groups/digitalisierung/-/boards) , [Support](https://git.muenchen.de/digitalisierung/digiwf-support) , [Zenhub](https://app.zenhub.com/workspaces/digiwf-621f70bf50ea1100120b7e93/board) )
- [ ] feature process is created or extended
- [ ] [git-ops](https://git.muenchen.de/digitalisierung/digiwf-ops) is customized
- [ ] Openshift environments are prepared (Secrets, etc.)

### CODE REVIEW

File: src/lib-components/VMultiFileInput.vue [hunk 1]
```diff
@@ -346,7 +361,7 @@ export default class VMultiFileInput extends Vue {
         filename,
         this.filePath
       );
-    } else {
+    } else if (this.formContext.type == "task") {
```
Comment (martind260) [inline src/lib-components/VMultiFileInput.vue:364]: @dominikhorn93 Why no download for the new context?

File: src/lib-components/VMultiFileInput.vue [hunk 2]
```diff
@@ -295,11 +299,16 @@ export default class VMultiFileInput extends Vue {
         this.formContext.id,
         this.filePath
       );
-    } else {
+    } else if (this.formContext.type == "task") {
       res = await HumanTaskFileRestControllerApiFactory(cfg).getFileNames(
         this.formContext.id,
         this.filePath
       );
+    } else {
```
Comment (martind260) [inline src/lib-components/VMultiFileInput.vue:307]: @dominikhorn93 What will the third type be?

Comment (dominikhorn93) [reply src/lib-components/VMultiFileInput.vue:307]: Added a comment 👍

File: src/lib-components/VMultiFileInput.vue (hunk 1 above)
Comment (dominikhorn93) [reply src/lib-components/VMultiFileInput.vue:366]: Copy & Paste mistake... thanks!

**Review by martind260 — APPROVED** [review]

**Pull request merged**